docs/design/coreclr/botr/readytorun-format.md

```diff
@@ -392,11 +429,26 @@ This section contains a native hashtable of all defined & export types within th
 |         0 | defined type
 |         1 | exported type
 
-The version-resilient hashing algorithm used for hashing the type names is implemented in [vm/versionresilienthashcode.cpp](https://github.com/dotnet/runtime/blob/8c6b1314c95857b9e2f5c222a10f2f089ee02dfe/src/coreclr/src/vm/versionresilienthashcode.cpp#L75).
+The version-resilient hashing algorithm used for hashing the type names is implemented in
+[vm/versionresilienthashcode.cpp](https://github.com/dotnet/runtime/blob/8c6b1314c95857b9e2f5c222a10f2f089ee02dfe/src/coreclr/src/vm/versionresilienthashcode.cpp#L75).
+
+**Note:** This is a per-assembly section. In single-file R2R files, it is pointed to directly by the
+main R2R header; in composite R2R files, each component module has its own available type section pointed to
+by the `READYTORUN_SECTION_ASSEMBLIES_ENTRY` core header structure.
 
 ## READYTORUN_SECTION_INSTANCE_METHOD_ENTRYPOINTS
 
-This section contains a native hashtable of all generic method instantiations compiled into the R2R executable. The key is the method instance signature; the appropriate version-resilient hash code calculation is implemented in [vm/versionresilienthashcode.cpp](https://github.com/dotnet/runtime/blob/master/src/coreclr/src/vm/versionresilienthashcode.cpp#L127); the value, represented by the `EntryPointWithBlobVertex` class, stores the method index in the runtime function table, the fixups blob and a blob encoding the method signature.
+This section contains a native hashtable of all generic method instantiations compiled into
+the R2R executable. The key is the method instance signature; the appropriate version-resilient
+hash code calculation is implemented in
+[vm/versionresilienthashcode.cpp](https://github.com/dotnet/runtime/blob/master/src/coreclr/src/vm/versionresilienthashcode.cpp#L127);
+the value, represented by the `EntryPointWithBlobVertex` class, stores the method index in the
+runtime function table, the fixups blob and a blob encoding the method signature.
+
+**Note:** In contrast to non-generic method entrypoints, this section is image-wide for
```

Comment:
> As mentioned elsewhere in this document, CoreCLR runtime requires changes to
properly look up methods stored in this section in the composite R2R case.

This should be in design notes.

Reply:
Good point, moved to design notes in 3rd commit.